shoulder_pan_joint: {p: 300.0, i: 0.0, d: 0.0}
shoulder_lift_joint: {p: 300.0, i: 0.0, d: 0.0}
elbow_joint: {p: 300.0, i: 0.0, d: 0.0}
wrist_1_joint: {p: 100.0, i: 0.0, d: 0.0}
wrist_2_joint: {p: 100.0, i: 0.0, d: 0.0}
wrist_3_joint: {p: 100.0, i: 0.0, d: 0.0}
robotiq_85_left_knuckle_joint: {p: 100.0, i: 0.0, d: 0.0}
